submissions/zhenyakornilov/memory-pair-game/js/index.js

```diff
@@ -0,0 +1,107 @@
+import cards from "./cards.js";
+
+const cardsMenu = document.getElementById("game-cards");
+const totalTriesCounter = document.getElementById("total-flips");
+const states = {
+  cardsMatches: 0,
+  firstFlippedCard: null,
+  secondFlippedCard: null,
+  currentCardPair: [],
+  totalTries: 0,
+};
+const allGameCards = shuffleCards([...cards, ...cards]);
```

Comment:
Seems allCards or shuffledCards will be enough here, to make it consistent with shuffleCards